source/solutions-library/credit-card-application-with-generative-ai.txt

```diff
@@ -0,0 +1,400 @@
+Credit Card Application with Generative AI 
+===========================================
+
+.. meta:: 
+   :keywords: AI, Credit, |service|, Spark, Vector Search
+   :description: Learn how the convergence of alternative data, artificial intelligence, and generative AI
+      is reshaping the foundations of credit scoring.
+
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Learn how the convergence of alternative data, artificial intelligence, and generative AI
+is reshaping the foundations of credit scoring.
+
+Solution Overview
+-----------------
+
+In this solution, you’ll learn how the convergence of alternative data, artificial
+intelligence, and `generative AI
+<https://www.mongodb.com/resources/basics/artificial-intelligence/generative-ai>`_. (gen
+AI) is reshaping the foundations of credit scoring. The challenges of traditional models
+are being overcome through the adoption of alternative credit scoring methods by offering
+a more inclusive and nuanced assessment of creditworthiness. This solution provides an
+example via an online credit card application process that illustrates an approach to
+realizing the transformative opportunities of gen AI and how |service| can be leveraged to
+supercharge credit scoring. This approach can be applied to other credit products — like
+personal loans, mortgages, corporate loans, and trade finance credit lines — and their
+applications without necessarily confining them to a credit card product only.
+
+.. video:: https://www.youtube.com/watch?v=tUgSCnGjTI0
+
+
+Challenges with Traditional Credit Scoring
+------------------------------------------
+
+The pursuit of credit can be a labyrinthine journey, particularly when credit assessment
+processes and, in particular, credit scoring mechanisms pose significant challenges. Here
+are some of the challenges or limitations of traditional credit scoring models:
+
+- **Limited credit history**: Many individuals encounter hurdles in the form of limited or
+  nonexistent credit history, making it difficult to prove their creditworthiness due to
+  the lack of historical data.
+
+- **Inconsistent income**: Irregular income, typical in part-time work or freelancing,
+  poses a challenge for traditional credit scoring models, which label individuals as
+  higher risk, leading to application denials or restrictive credit limits.
+
+- **High utilization of existing credit**: Heavy reliance on existing credit, leading to
+  elevated credit utilization ratios, becomes a stumbling block in credit applications as
+  applicants may face rejection or approval with less favorable terms.
+
+- **Lack of clarity in rejection reasons**: A lack of transparency in rejection reasons
+  leaves applicants in the dark, making it difficult for them to address the root cause
+  and enhance their creditworthiness for future applications.
+
+Transforming Credit Scoring and Applications with |service|
```

Comment:
Another substitution that comes out as "Atlas" instead of "MongoDB". I'd suggest reviewing all the uses in this article to make sure they're showing up with the right text on staging.